### PR TITLE
Improve explanation how join works with duplicates

### PR DESCRIPTION
**Pull request by bkamins**
Following the discussion on Slack.

### CODE REVIEW

**Comment by bkamins** [comment]
@oxinabox - I have passed @nalimilan biological CI 😄. If the description is clear for you we can merge it.

**Comment by oxinabox** [comment]
I think the explaination should be cross-referenced with the `validate` kwarg.
Either referencing back to it, or it referencing forward to the example.

**Comment by bkamins** [comment]
Thank you. Actually the documentation was more outdated than I thought. I have cleaned it up a bit in general.

File: docs/src/man/joins.md [hunk 1]
```diff
@@ -219,3 +200,41 @@ julia> join(a, b, on = [:City, :Job])
...
+
+```jldoctest joins
+julia> join(a, b, on = [(:City, :Location), (:Job, :Work)], validate=(true, true))
+ERROR: ArgumentError: Merge key(s) are not unique in both df1 and df2. First duplicate in df1 at 3. First duplicate in df2 at 3
+```
+
+Finally, using `indicator` keyword argument you can add a column to the resulting data frame giving an information if the given row appeared only in the left, the right or both data frames. Here is an example:
+
+```jldoctest joins
+julia> a = DataFrame(ID = [20, 40], Name = ["John", "Jane"])
```
Comment (nalimilan) [inline docs/src/man/joins.md]: Can't we use the existing data frames here?

Comment (bkamins) [reply docs/src/man/joins.md]: No, because in the existing data frames all keys match so we would see `"both"` everywhere.

**Pull request merged**